tensorrt_llm/serve/openai_disagg_server.py

```diff
@@ -39,8 +40,11 @@ def __init__(self,
         self.ctx_server_idx = 0
         self.gen_server_idx = 0
 
-        if (len(self.ctx_servers) == 0) or (len(self.gen_servers) == 0):
-            raise ValueError("At least one context server and one generation server must be provided")
+        if (len(self.gen_servers) == 0):
+            raise ValueError("At least one generation server must be provided")
+
+        if os.getenv("TRTLLM_DISAGG_BENCHMARK_GEN_ONLY") != "1" and len(ctx_servers) == 0:
```

Comment:
I think we need to provide a default value here to make sure the code doesn't break when the env is not set.